pypeclub/OpenPype
# Webpublisher: Fix use ftrack username in burnins

## Brief description
Burnins were using default `pypeclub` username

## Description
If ftrack username is found based on provided user email, it is used in burnins (left bottom corner)

## Testing notes:
1. Publish via Webpublish with user email that doesn't map to default `pypeclub` username
2. Check bottome left corner of published burnin
